slides/rslondon-southeast-july-2023/index.qmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Superseding R packages - lessons learned from transitioning to {epichains} from {bpmodels}"
+author: 
+  - name: "James M. Azam"
+    orcid: "0000-0001-5782-7330"
+date: "2023-07-17"
+categories: [Epiverse-TRACE, workshop, community, r-packages]
```

Comment:
```suggestion
categories: [workshop, community, R package, bpmodels, epichains]
```

The "Epiverse-TRACE tag is unnecessary" since this is the Epiverse-TRACE blog. I'll submit a PR to fix it in the Harmonize page as well.

Reply:
Is it not necessary for the wider web?

Reply:
No, it's not used in the page metadata